docs/getting_started/quickstart.rst

```diff
@@ -165,6 +165,11 @@ across a single column via the ``apply`` operation.
 Modin is more than 30X faster at applying a single column of data, operating on 130+ 
 million rows in a second.
 
+In short, Modin provides orders of magnitude speed up over pandas for a variety of operations out of the box. 
+
+.. figure:: ../img/quickstart_speedup.svg
+   :align: center
+
```

Comment:
Do we want to have some proof of this? For example, a link to the reproducer & the data being used. I believe such speedups are not possible on any of the data & parameters combinations for presented functions.

The speed-up is impressive and it would be even more if we put a reproducer so any user could run it on their machines and ensure by themselves.